blocks/shared-styles/_children/promo-image/index.test.jsx

```diff
@@ -0,0 +1,135 @@
+import React from 'react';
+import { mount } from 'enzyme';
+
+import PromoImage from './index';
+
+jest.mock('fusion:properties', () => (jest.fn(() => ({
+  resizerURL: 'https://resizer.me',
+}))));
+
+jest.mock('fusion:content', () => ({
+  useEditableContent: jest.fn(() => ({
+    editableContent: () => ({ contentEditable: 'true' }),
+  })),
+  useContent: jest.fn(() => ({
+    '800x600': 'VWgB9mYQ5--6WT0lD6nIw11D_yA=filters:cm=t/',
+    '600x450': 'nMHJLqy4jBawvstr8mcTCfgBzoE=filters:cm=t/',
+    '400x300': 'D9yXcc8gxtvlCODv62qbg2i52N8=filters:cm=t/',
+  })),
+}));
+
+jest.mock('fusion:context', () => ({
+  useFusionContext: jest.fn(() => ({
+    arcSite: 'the-sun',
+  })),
+}));
+
+jest.mock('fusion:themes', () => (jest.fn(() => ({}))));
+
+describe('PromoImage', () => {
+  it('renders PlaceholderImage if nothing passed in', () => {
+    const wrapper = mount(<PromoImage />);
+
+    expect(wrapper.find('PlaceholderImage').exists()).toBe(true);
+  });
+
+  it('renders custom image without a link', () => {
+    const props = {
+      customImageURL: 'https://d22ff27hdsy159.cloudfront.net/12-17-2019/t_b47df64ea45e4b178442347c0dc3724b_name_file_1280x720_2000_v3_1_.jpg',
```

Comment:
this doesn't have to be a valid link. in fact, for costs, I'd almost prefer it not to be a legit image

Reply:
I usually do something like 'https://fake-image.jpg' or something obviously not valid